### PR TITLE
FINERACT-2076: SQL query optimization

### PR DESCRIPTION
**Pull request by vidakovic**
Improve SQL queries.

### CODE REVIEW

File: fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/DefaultSqlValidator.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import lombok.extern.slf4j.Slf4j;
+import org.apache.fineract.infrastructure.core.config.FineractProperties;
+import org.apache.fineract.infrastructure.security.exception.SqlValidationException;
+import org.apache.fineract.infrastructure.security.service.SqlValidator;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+// @ConditionalOnMissingBean(SqlValidator.class)
```
Comment (adamsaghy) [inline fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/DefaultSqlValidator.java]: I guess we dont need this commented out conditional.

**Review by adamsaghy — COMMENTED** [review]
LGTM

File: fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/DefaultSqlValidator.java (hunk 1 above)
Comment (vidakovic) [reply fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/DefaultSqlValidator.java]: Can't hide anything ;-) ... still working on that... want to make this replaceable.

Comment (vidakovic) [reply fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/DefaultSqlValidator.java]: Removed it

**Review by adamsaghy — APPROVED** [review]
LGTM

**Pull request merged**